src/unix/syscall.c

```diff
@@ -2433,15 +2467,47 @@ static void syscall_schedule(context f)
     syscall_debug(f);
 }
 
-void init_syscalls()
+static char *missing_files_exclude[] = {
+    "ld.so.cache",
```

Comment:
Just curious - why is it necessary to exclude ld.so.cache - or any files for that matter?

Reply:
The intent was to reduce confusion for a user who may not understand what a file does, especially since ld.so.cache is generated for the host and I was concerned it might be a source of more trouble if a copied ld.so.cache had references to files or paths which did not exist on the nanos image. I'm happy to take it out if you think it's better to just show everything.

Reply:
No need to change - just curious!